main.go

```diff
@@ -1,126 +1,126 @@
 package main
 
 import (
-	"log"
-	"net/http"
-
-	"fmt"
-
-	"strings"
-
-	"github.com/jckuester/weather-station/arduino"
-	"github.com/jckuester/weather-station/pulse"
+	"context"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"gopkg.in/alecthomas/kingpin.v2"
+	"log"
+	"net/http"
+	"strings"
 )
 
 var (
 	device = kingpin.Flag("device", "Arduino connected to USB").
 		Default("/dev/ttyUSB0").String()
 	listenAddr = kingpin.Flag("listen-address", "The address to listen on for HTTP requests.").
 			Default(":8080").String()
-	ids = kingpin.Arg("ids", "Sensor IDs that will be exported").Ints()
+	ids = kingpin.Arg("ids", "Sensor IDs that will be exported").StringMap()
```

Comment:
I think there could be better ways to do the config. It was just the most straigh forward way for now

Reply:
Sure, for example to load new sensors without stopping/restarting the exporter.

Reply:
this might be done with a config file and a file watcher automatically reading the new values into memory. for example.. but it's not the most critical thing IMO

Reply:
Agree, not critical.